email_routing_destination.go

```diff
@@ -0,0 +1,142 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+type EmailRoutingDestinationAddress struct {
+	Tag      string     `json:"tag,omitempty"`
+	Email    string     `json:"email,omitempty"`
+	Verified *time.Time `json:"verified,omitempty"`
+	Created  *time.Time `json:"created,omitempty"`
+	Modified *time.Time `json:"modified,omitempty"`
+}
+
+type EmailRoutingListAddressParameters struct {
+	ResultInfo
```

Comment:
we have a couple of other filtering options here too

```suggestion
	ResultInfo
	Direction string `url:"direction,omitempty"`
	Verified  *bool  `url:"verified,omitempty"`
```